synapse/models/inet.py

```diff
@@ -960,6 +966,13 @@ def getBaseModels():
                          'doc': 'The optional password used to access the URL.'}),
                 ]),
 
+                ('inet:urlredir', {}, [
+                    ('src', {'ptype': 'inet:url', 'ro': 1, 'req': 1,
+                        'doc': 'The original/source URL before redirect'}),
+                    ('dst', {'ptype': 'inet:url', 'ro': 1, 'req': 1,
+                        'doc': 'The redirected/destination URL'}),
+                ]),
```

Comment:
Suggest adding optional secondary props seen:min and seen:max

Reply:
add src:domain and dst:domain